mock/cpp_mock_scenarios/src/cpp_scenario_node.cpp

```diff
@@ -86,7 +95,7 @@ void CppScenarioNode::stop(Result result, const std::string & description)
 }
 
 void CppScenarioNode::spawnEgoEntity(
-  const traffic_simulator::CanonicalizedLaneletPose & spawn_lanelet_pose,
+  const traffic_simulator::LaneletPose & spawn_lanelet_pose,
```

Comment:
Please use traffic_simulator::CanonicalizedLaneletPose, functions in API class should use traffic_simulator::CanonicalizedLaneletPose

```suggestion
  const traffic_simulator::CanonicalizedLaneletPose & spawn_lanelet_pose,
```

Reply:
@hakuturu583 
Please take a look at [this comment ](https://github.com/tier4/scenario_simulator_v2/pull/1266#discussion_r1643476928)- now the conversion from `msg::LaneletPose` to `CanonicalziedLaneletPose` is the responsibility of `EntityManager::spawnEntity()` - which has access to all the needed data, so there is no need to require `pose::toCanonicalziedLaneletPose()`/`pose::canonicalize()` call before calling `spawn()`.